gstreamer/src/tags.rs

```diff
@@ -554,6 +570,179 @@ impl StaticType for TagListRef {
     }
 }
 
+#[cfg(feature = "serde-support")]
+struct TagSer<'a>(&'a str, Value);
+
+#[cfg(feature = "serde-support")]
+impl<'a> Serialize for TagSer<'a> {
+    fn serialize<S: Serializer>(&self, serializer: S) -> Result<S::Ok, S::Error> {
+        let mut tup = serializer.serialize_tuple(2)?;
+        tup.serialize_element(self.0)?;
+        match self.1.type_() {
+            // TODO: use a macro serialize_tag_value!
+            glib::Type::String => tup.serialize_element(&self.1.get::<String>().unwrap())?,
+            glib::Type::U32 => tup.serialize_element(&self.1.get::<u32>().unwrap())?,
```

Comment:
There's also `gst::ValueExt::serialize` and the inverse which could be useful here